src/components/trashlookup.jsx

```diff
@@ -1,156 +1,121 @@
-import PostData from "../Data/street.json";
-import SearchBox from "./searchBox";
-import _ from "lodash";
-import ListGroup from "./listGroup";
-import Trash from "./common/trash";
-import Leaf from "./common/leaf";
-import Recycle from "./common/recycle";
-import { getTrashService } from "../services/trashService";
+import React from 'react';
+import Form from './common/Form';
+import PostData from '../Data/street.json';
+import _ from 'lodash';
+import InformationSection from './common/InformationSection';
+import { getTrashService } from '../services/trashService';
+import * as moment from 'moment';
+import ScheduleTable from './common/scheduleTable';
+class TrashLookUp extends Form {
+  displayCollectionSechduleText() {
+    const selectedAddress = _.trim(this.state.selectedAddress);
+    return <InformationSection address={selectedAddress} resetForm={this.handleClick} />;
+  }
 
-class Movies extends Component {
-  state = {
-    services: getTrashService(),
-    searchQuery: "",
-    addresses: PostData,
-    issearch: 0,
-    selectedAddress: null
-  };
+  addressData() {
+    let searchQuery = _.trim(this.state.selectedAddress);
+    let filtered = PostData;
+    const isCollectionScheduleTextHidden = this.state.isCollectionScheduleTextHidden;
 
-  // componentDidMount() {
-  // console.log("inside didmount");
-  //const fakeAddress = [{ _id: "", name: "" }, ...getFakeAddress()];
-  // this.setState({ searchQuery: this.ref.searchboxinput.value });
-  // }
-  // componentDidUpdate() {
-  //   console.log("inside componentDidUpdate" + this.state.searchQuery);
-  // }
-  handleAddressSelect = address => {
-    // console.log("address:" + address.address1);
-    this.setState({
-      selectedAddress: address.address1,
-      searchQuery: address.address1,
-      issearch: 3
-    });
-  };
-  handleSearch = query => {
-    console.log("handel Search:" + query);
-    if (query.length === 0) {
-      this.setState({ searchQuery: query, issearch: 0 });
-    } else {
-      this.setState({ searchQuery: query, issearch: 1 });
+    if (searchQuery.length > 0) {
+      if (isCollectionScheduleTextHidden) {
+        filtered = PostData.filter(m => m.address1.toLowerCase().indexOf(searchQuery.toLowerCase()) > -1);
+      } else {
+        filtered = PostData.filter(m => m.address1.toLowerCase() === searchQuery.toLowerCase());
+      }
     }
-  };
+    return { data: filtered };
+  }
+  trashSchedule() {
+    const scheduleType = this.state.scheduleType;
+    //TODO: we will use this later
+    // const dowloadMessage =
+    //   "Download your complete four year scheduled in PDF format";
+    return (
+      <div className="row">
+        <div className="col-5">
+          <ScheduleTable
+            services={getTrashService()}
+            scheduleType={scheduleType}
+            renderWeekOfDay={this.renderWeekOfDay}
+          />
+          <h6>Download</h6>
+          {/* <a href="/test">{dowloadMessage}</a> */}
```

Comment:
please add a comment why this is commented out, something like `this feature isn't enabled, but will be in the future` is sufficient

Reply:
similar to waht you did above.

Reply:
added comments